ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -1120,4 +1120,55 @@ ActorScope,string,Optional,ProcessEvent,,,
 TargetOriginalUserType,string,Optional,ProcessEvent,,,
 TargetScopeId,string,Optional,ProcessEvent,,,
 TargetScope,string,Optional,ProcessEvent,,,
-
+User,string,Alias,UserManagement,,,ActorUserName
+Hostname,string,Alias,UserManagement,,,DvcHostname
+IpAddr,string,Alias,UserManagement,,,ScrIpAddr
+UpdatedPropertyName,string,Alias,UserManagement,,MultipleProperties|UpdatedPropertyName,EventSubType
+ActorUserType,string,Optional,UserManagement,Enumerated,Regular|Machine|Admin|System|Application|Service Principal|Other,
+ActorUsernameType,string,Conditional,UserManagement,Enumerated,UPN|Windows|DN|Simple,ActorUsername
+EventType,string,Mandatory,UserManagement,Enumerated,UserCreated|UserDeleted|UserModified|UserLocked|UserUnlocked|UserDisabled|UserEnabled|PasswordChanged|PasswordReset|GroupCreated|GroupDeleted|GroupModified|UserAddedToGroup|UserRemovedFromGroup|GroupEnumerated|UserRead|GroupRead,
+EventResult,string,Mandatory,UserManagement,Enumerated,Success|Failure,
+EventSeverity,string,Mandatory,UserManagement,Enumerated,Informational|Low|Medium|High,
+EventSchema,string,Mandatory,UserManagement,,UserManagement,
+EventSchemaVersion,string,Mandatory,UserManagement,SchemaVersion,,
+ActingAppId,string,Optional,UserManagement,,,
+ActiveAppName,string,Optional,UserManagement,,,
+ActingAppType,string,Optional,UserManagement,Enumerated,Process|Browser|Resource|Other,
+HttpUserAgent,string,Optional,UserManagement,Useragent,,
+ActorUsername,string,Mandatory,UserManagement,Username,,
+ActorUserIdType,string,Conditional,UserManagement,Enumerated,SID|UID|AADID|OktaId|AWSId,ActorUserId
+ActorUserId,string,Optional,UserManagement,,,
+ActorSessionId,string,Optional,UserManagement,,,
+EventSubType,string,Optional,UserManagement,Enumerated,UserRead|UserCreated|GroupCreated|UserModified|GroupModified,
```

Comment:
This should have three values for now, "Password", "Hash" and "MultipleProperties". We can add remaining in future.

Reply:
updated, altough is does not fully reflect the documentation:
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-user-management#eventsubtype:~:text=UserRead%3A%20Password%2C%20Hash%0A%2D%20UserCreated%2C%20GroupCreated%2C%20UserModified%2C%20GroupModified